### ISSUE TITLE
HTML5 Povider - live + pdt 

### ISSUE THREAD

**Issue by mtoczko** [jwplayer/jwplayer#3519]
### Expected Behavior

No errors in metadata handling (PROGRAM-DATE-TIME)

### Actual Behavior
Live stream + pdt (playlist duration 30s)
```
#EXT-X-VERSION:4
#EXT-X-TARGETDURATION:10
#EXT-X-MEDIA-SEQUENCE:173
#EXT-X-PROGRAM-DATE-TIME:2019-11-04T14:01:55.565+00:00
#EXTINF:10.0,
media-u0gia6ek1_b2228224_slpl_173.ts
#EXTINF:10.0,
media-u0gia6ek1_b2228224_slpl_174.ts
#EXTINF:10.0,
media-u0gia6ek1_b2228224_slpl_175.ts
```
**console**
```
TypeError: The provided value is non-finite

```
**function**
```
createCue: function(e, t, i) {
                var n = window.VTTCue || window.TextTrackCue,
                    o = Math.max(t || 0, e + .25);
                return new n(e, o, i)
```
```
e: 0
i: "{\"metadataType\":\"program-date-time\",\"programDateTime\":\"2019-11-04T14:55:46.345Z\",\"start\":0,\"end\":null}"
n: function()
o: Infinity
t: Infinity
```
When the playlist duration is longer than 30s, there is no error in the console.
```
#EXT-X-VERSION:4
#EXT-X-TARGETDURATION:10
#EXT-X-MEDIA-SEQUENCE:173
#EXT-X-PROGRAM-DATE-TIME:2019-11-04T14:01:55.565+00:00
#EXTINF:10.0,
media-u0gia6ek1_b2228224_slpl_173.ts
#EXTINF:10.0,
media-u0gia6ek1_b2228224_slpl_174.ts
#EXTINF:10.0,
media-u0gia6ek1_b2228224_slpl_175.ts
#EXTINF:10.0,
media-u0gia6ek1_b2228224_slpl_176.ts
```
**Proposed solution**
https://github.com/jwplayer/jwplayer/blob/4bf733f143537f0c6c3ff5f7fad34a572cded015/src/js/providers/tracks-mixin.js#L47

```
const cueEnd = Math.max( isFinite(end) ? end : 0, start + 0.25); 
```

### Environment

- *operating system | version* macOS 10.14.6
- *browser | version* Safari  13.0.1
- *JW Player version | url* 8.11.5

**Comment by karimMourra** [comment]
@mtoczko thanks for sharing this! Could you please share your test stream ?

**Comment by mtoczko** [comment]
@karimMourra 
I can't share 24/7 broadcast. I can write a simulation. But it will take a few days. Don't have your own test streams?

**Comment by karimMourra** [comment]
@mtoczko we have our own test streams, but none reproducing the issue you describe

**Comment by mtoczko** [comment]
@karimMourra  
test stream: [hls-pdt.zip](https://github.com/jwplayer/jwplayer/files/3824369/hls-pdt.zip)

**Comment by karimMourra** [comment]
thank you for sharing. Are you seeing this issue in the latest JWPlayer Version ? I notice you are reporting this issue for version 7.11.5, but we are no longer patching version 7. 
Our latest version is 8.11.7 and we strongly encourage you to upgrade. If you cannot upgrade to version 8, the last version of JW7 was 7.12.13

**Comment by mtoczko** [comment]
I'm sorry my mistake. I meant version 8.11.5/8.11.7

**Comment by karimMourra** [comment]
@mtoczko a ticket was created in our backlog and will be prioritized soon. We will keep you updated with the progress. Thank you for your contribution!

**Comment by robwalch** [comment]
@mtoczko This will be fixed in v8.13 with #3620. Thanks for the bug report!
